Makefile

```diff
@@ -15,13 +15,6 @@ libpak-tools: out
 	@echo "> Building libpak-tools..."
 	go build -ldflags=$(LDFLAGS) -o $(OUTDIR)/libpak-tools main.go
 
-package: OUTDIR=./bin
```

Comment:
This is removed as it's left over. We don't need it any more with goreleaser in place.